scripts/zombienet/config-network-a.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
I think we need to add `--pruning=archive` to the configs

Reply:
Done.